README.adoc

```diff
@@ -147,4 +153,4 @@ Hawkular-Alerts is released under Apache License, Version 2.0 as described in th
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```

Comment:
huh, not sure what happened on this line. I used Vim to edit just the top of the file.